core/vibe/core/drivers/libevent2.d

```diff
@@ -1086,6 +1103,34 @@ final class Libevent2UDPConnection : UDPConnection {
 		}
 	}
 
+	void addMembership(string group)
```

Comment:
Makes sense. Do you want me to squash and force update? Or create additional commit?